argocd/overlays/moc-infra/projects/otel-dev.yaml

```diff
@@ -0,0 +1,26 @@
+apiVersion: argoproj.io/v1alpha1
+kind: AppProject
```

Comment:
@sallyom can you also include `otel-dev.yaml` project [here](https://github.com/operate-first/apps/blob/master/argocd/overlays/moc-infra/projects/kustomization.yaml)?

Reply:
done!